docarray/store/helpers.py

```diff
@@ -167,13 +183,28 @@ def _to_binary_stream(
 
 
 def _from_binary_stream(
+    self, protocol: Literal['pickle', 'protobuf'],
```

Comment:
same here we can do like you id in the other

Reply:
<img width="1186" alt="image" src="https://github.com/docarray/docarray/assets/81156510/90ed897a-0890-496b-aeb1-00aaf9543d25">

Facing this error When I try to commit the changes. Any help on the same appreciated.

Reply:
@samsja Can you have a look at this? Meanwhile let me work on the function implementation.